src/library/browse/browsefeature.cpp

```diff
@@ -469,8 +478,18 @@ std::vector<std::unique_ptr<TreeItem>> BrowseFeature::getChildDirectoryItems(
     QFileInfoList all = dirAccess.info().toQDir().entryInfoList(
             QDir::Dirs | QDir::NoDotAndDotDot);
 
+    int count = 0;
+
     // loop through all the item and construct the children
-    foreach (QFileInfo one, all) {
+    // foreach (QFileInfo one, all) {
+
+    for (const QFileInfo& one : all) {
```

Comment:
to fix the clazy warning:
```suggestion
    for (const QFileInfo& one : std::as_const(all)) {
```